spec/support/integration_helpers.rb

```diff
@@ -14,7 +14,7 @@ def run_rspec(path, chdir: nil, success: true, env: {})
   def run_minitest(path, chdir: nil, success: true, env: {})
     output, status = Open3.capture2(
       env,
-      "ruby #{path}_fixture.rb",
+      "ruby #{path}_fixture.rb #{env['TESTOPTS']}",
```

Comment:
Shouldn't it work without passing explicitly as arg? I mean, having `TESTOPTS` env variable should just work

Reply:
It will work perfectly when running test via `rake`. Running test via Ruby interpreter accepts test name as CLI option. That's why it's here.

Reply:
Ok